sdks/java/harness/jmh/src/main/java/org/apache/beam/fn/harness/jmh/control/ExecutionStateSamplerBenchmark.java

```diff
@@ -103,33 +103,37 @@ public void tearDown() {
   }
 
   @Benchmark
-  @Threads(1)
-  public void testTinyBundleRunnersCoreStateSampler(RunnersCoreStateSampler state)
+  @Threads(10)
+  public void testTinyBundleRunnersCoreStateSampler(RunnersCoreStateSampler state, Blackhole bh)
       throws Exception {
-    state.tracker.activate();
+    ExecutionStateTracker tracker = new ExecutionStateTracker(state.sampler);
```

Comment:
We should create a JMH `@State` called `RunnersCoreStateTracker` at thread scope. Then in the setup method take in a parameter of the RunnersCoreStateSampler.

You can see an example of what I mean here: https://github.com/openjdk/jmh/blob/1f2befef92c3eb1466124f37d00f496b4105d3c5/jmh-samples/src/main/java/org/openjdk/jmh/samples/JMHSample_29_StatesDAG.java#L134

This will allow us to move the initialization outside of the benchmark into state. 

Ditto on creating `HarnessStateTracker` as well.